src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
```

Order - as elements appear on the page
```suggestion
html {
  font-size: 12px;
  font-family: "Roboto", sans-serif;
  font-weight: 500;
}

body {
  margin: 0;
}
```